nengo/spa/compare.py

```diff
@@ -52,16 +47,15 @@ def __init__(self, dimensions, vocab=None, neurons_per_multiply=200,
 
         with self:
             nengo.Connection(self.inputA,
-                             self.compare.input[::2], synapse=None)
+                             self.product.A, synapse=None)
             nengo.Connection(self.inputB,
-                             self.compare.input[1::2], synapse=None)
-            self.compare.add_output('product', lambda x: x[0] * x[1])
+                             self.product.B, synapse=None)
 
     def on_add(self, spa):
```

Comment:
Is this function necessary now? Originally it was done on add because a vocabulary was being used (I believe)